ftest/webdriver/pom.xml

```diff
@@ -18,6 +18,18 @@
     <some.password>setTheValueInYourSettings</some.password>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-jsf-ui</artifactId>
+        <version>${nuxeo.distribution.version}</version>
```

Comment:
You can use `${project.version}` here since it is the same as the distribution one, same for the others.

Reply:
done ;)